default.nix

```diff
@@ -4,14 +4,45 @@ with import (builtins.fetchGit {
   rev = "95fed28ac372c61eb83c87ad97c24b0f957827bf";
 }) {};
 
+let
+  build =
+    writeShellScriptBin "postgrest-docs-build"
+      ''
+        set -euo pipefail
+
+        sphinx-build -b html -a -n . _build
+      '';
+
+
+  serve =
+    writeShellScriptBin "postgrest-docs-serve"
+      ''
+        set -euo pipefail
+
+        python livereload_docs.py
+      '';
+
+
+  spellcheck =
+    writeShellScriptBin "postgrest-docs-spellcheck"
+      ''
+        set -euo pipefail
+
+        cat *.rst | grep -v '^\(\.\.\|  \)' | sed 's/`.*`//g' | aspell -d ${aspellDicts.en}/lib/aspell/en_US -p ./postgrest.dict list | tee misspellings
```

Comment:
In theory I would need to prefix `cat`, `grep` etc. as well, right? Am I going to do that - or where do I draw the cut?

Reply:
A few standard utilities are on the path by default, see https://nixos.org/manual/nixpkgs/stable/#sec-tools-of-stdenv . You can easily test whether you got it right by using `nix-shell --pure ...`

Reply:
Ah `--pure` is nice. That helps a lot. Works!